### PR TITLE
[SMALLFIX] doc improvement for location policy

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

**Comment by yupeng9** [comment]
Jenkins, test this please

File: docs/en/File-System-API.md [hunk 1]
```diff
@@ -82,30 +82,32 @@ Below is a table of the expected behaviors of `WriteType`
 
 ### Location policy
 
-Alluxio provides location policy to choose which workers to store the blocks of a file. User can set
-the policy in `CreateFileOptions` for writing files and `OpenFileOptions` for reading files into
-Alluxio. Alluxio supports custom location policy, and the built-in polices include:
+Alluxio provides location policy to choose which workers to store the blocks of a file. 
 
-* **LocalFirstPolicy**
+Using Alluxio's Java API, user can set the policy in `CreateFileOptions` for writing files and `OpenFileOptions` for reading files into
```
Comment (gpang) [inline docs/en/File-System-API.md]: user -> users

File: docs/en/File-System-API.md [hunk 2]
```diff
@@ -82,30 +82,32 @@ Below is a table of the expected behaviors of `WriteType`
...
+Alluxio.
+
+Or to use the built-in policies, user can also simply override the default policy class in the [configuration file](Configuration-Settings.html) at property `alluxio.user.file.write.location.policy.class`. The built-in polices include:
```
Comment (gpang) [inline docs/en/File-System-API.md]: remove: "Or to use the built-in policies, "
user can also -> Users can
polices  -> policies

**Comment by gpang** [comment]
I left a few comments. Thanks!

**Comment by yupeng9** [comment]
Thanks for the review. Addressed the comments.

**Comment by gpang** [comment]
LGTM

**Pull request merged**